src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -888,6 +888,28 @@ public abstract partial class JsonValue : System.Text.Json.Nodes.JsonNode
         public abstract bool TryGetValue<T>([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] out T? value);
     }
 }
+namespace System.Text.Json.Schema
+{
+    public static partial class JsonSchemaExporter
+    {
+        public static System.Text.Json.Nodes.JsonNode GetJsonSchemaAsNode(this System.Text.Json.JsonSerializerOptions options, System.Type type, System.Text.Json.Schema.JsonSchemaExporterOptions? exporterOptions = null) { throw null; }
+        public static System.Text.Json.Nodes.JsonNode GetJsonSchemaAsNode(this System.Text.Json.Serialization.Metadata.JsonTypeInfo typeInfo, System.Text.Json.Schema.JsonSchemaExporterOptions? exporterOptions = null) { throw null; }
```

Comment:
Remind me why we went with the "AsNode" suffix? Is that because we expect to have a schema type in the future and we want GetJsonSchema in the future to return that?

Reply:
Correct.